frontend/components/nip46/Requests.tsx

```
@@ -43,6 +44,14 @@ const formatTimestamp = (value: string) => {
   return Number.isNaN(date.getTime()) ? 'N/A' : date.toLocaleString()
 }
 
+const sanitizePreview = (value: string): string => {
+  return value
+    .replace(/[\u0000-\u001F\u007F-\u009F]/g, '')
```

Biome (2.4.4)

[error] Unexpected control character in a regular expression.

(lint/suspicious/noControlCharactersInRegex)